test/navigation/navigators/BeneficiaryRoutes.scala

```diff
@@ -193,6 +193,24 @@ trait BeneficiaryRoutes {
       }
     }
 
+    "go to IndividualBeneficiaryAddressUKPage from IndividualBeneficiaryAddressUKYesNoPage when user answers yes" in {
+      forAll(arbitrary[UserAnswers]) {
+        userAnswers =>
+          val answers = userAnswers.set(IndividualBeneficiaryAddressUKYesNoPage(indexForBeneficiary), value = true).success.value
+          navigator.nextPage(IndividualBeneficiaryAddressUKYesNoPage(indexForBeneficiary), NormalMode)(answers)
+            .mustBe(routes.IndividualBeneficiaryAddressUKController.onPageLoad(NormalMode, indexForBeneficiary))
+      }
+    }
+
+    "go to IndividualBeneficiaryAddressUKPage from IndividualBeneficiaryAddressUKPage when user answers no" in {
```

Comment:
It should be `go to IndividualBeneficiaryAddressUKYesNoPage`?

Reply:
Amended